static/docs/tutorials/pipelines.md

```diff
@@ -38,11 +38,12 @@ $ git add code/
 $ git commit -m "Download and add code to new Git repo"
 ```
 
-> `dvc get` can download data artifacts from any DVC project hosted on a Git
-> repository into the current working directory (similar to `wget` but for DVC
-> repositories). In this case we use our own
-> [iterative/dataset-registry](https://github.com/iterative/dataset-registry))
-> project as the external data source.
+> `dvc get` can download <abbr>data artifacts</abbr> from any <abbr>DVC
```

Comment:
I got a feedback that people might think that data is actually stored in Git and `dvc get` just gets it from here. It can spook some professionals if they think that we put data into Git. Can we try to come with a better term than "data source" in this case to emphasize that git repo is just a _key_ to the data, not a storage by itself?

Reply:
Re-worded in 45fb574d. What do you think?

Reply:
It's not better since it's still `download from hosted on Git thing` ... may be something like: `dvc get` downloads data using Git repository (that is a DVC project) as a "pointer" to a specific data artifact version and its location

Reply:
It's "download data artifacts **from the remote storage** of any DVC project hosted on a Git repository" now, actually. Should be clear enough if read carefully, but I guess the sentence is too long/indirect. Will improve, trying to use your suggested wording...

Reply:
hmm 🤔 I don't see the ` from the remote storage ` part. Am I missing something?

Reply:
It's under https://dvc-org-pr-679.herokuapp.com/doc/tutorials/pipelines#preparation: 

![image](https://user-images.githubusercontent.com/1477535/68324993-913cd980-0096-11ea-997a-148d992f886f.png)

i'm still planning to change this again though...

Reply:
Updated. Please check.